lib/daos/driversDao.js

```diff
@@ -0,0 +1,29 @@
+import { drivers } from 'models';
+
+const attributes = [
+    'driver_id',
+    'cab_id',
```

Comment:
```suggestion
    'id',
```

Reply:
Right, will fix all the renaming issues.